mongo/crud_examples_test.go

```diff
@@ -0,0 +1,128 @@
+// Copyright (C) MongoDB, Inc. 2017-present.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License. You may obtain
+// a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+package mongo_test
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+// Client examples
+
+func ExampleClient_ListDatabases() {
+	var client *mongo.Client
+
+	// use a filter to only select the admin database
+	// specify the NameOnly option so the returned specificiations only have the name field populated
+	opts := options.ListDatabases().SetNameOnly(true)
```

Comment:
Seems sort of redundant with ListDataBaseNames.  Not sure why we want to show something for which there's a better way to do it.  Maybe loop and print name and sizeOnDisk instead?